src/prommis/HD_CE.py

```diff
@@ -0,0 +1,344 @@
+from pyomo.environ import ConcreteModel, Constraint, Param, Var, units,PositiveReals, log
+from pyomo.environ import units as pyunits
+from idaes.core import FlowsheetBlock,UnitModelCostingBlock,UnitModelBlock
+from idaes.models.costing.SSLW import SSLWCosting, SSLWCostingData, VesselMaterial
+from idaes.core.solvers import get_solver
+from math import pi 
```

Comment:
For consistency, you should get do:

```
from idaes.core.util.constants import Constants
```

You can then get `pi` as `Cosntants.pi` (along with a number of other useful constants).